auth/auth-router.js

```diff
@@ -18,7 +18,9 @@ router.post("/register", (req, res) => {
 
   Users.add(user)
     .then(saved => {
-      res.status(201).json(saved);
+      const token = generateToken(user);
+      console.log("token", token)  
```

Comment:
One trick for slightly shorter/quicker console logs I like to use is objects. Since you can use the name only once if you have a variable with the same name as a key, you can do it this way:
```javascript
console.log({ token });
```